css/templates/travel/page-vars.css

```diff
@@ -15,6 +15,22 @@
  */
 
 :root {
+  --blue:  #3682EC;
```

Comment:
Some of these may conflict with the code added in https://github.com/ampproject/ampstart/pull/666/files, might be a worth a rebase

Reply:
ok good call. Thanks for looking this over!